scripts/migrate_from_gmaven.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+set -euo pipefail
+
+# Replace the load statements
+find ./ -type f -name BUILD* -exec sed -i 's/gmaven_rules/rules_jvm_external/g' {} \;
```

Comment:
For all of these commands, is it possible people have these rules in bzl files?

Reply:
Yes, but I don't think this rule should cover them. It's quite basic, but we can add more flexible features in follow up PRs.